dll/win32/comctl32/tooltips.c

```diff
@@ -819,7 +819,7 @@ TOOLTIPS_Show (TOOLTIPS_INFO *infoPtr, BOOL track_activate)
         hRgn = CreateRoundRectRgn(0,
                                   (infoPtr->bToolBelow ? BALLOON_STEMHEIGHT : 0),
                                   rect.right - rect.left,
-                                  (infoPtr->bToolBelow ? rect.bottom - rect.top : rect.bottom - rect.top - BALLOON_STEMHEIGHT),
+                                  (infoPtr->bToolBelow ? rect.bottom - rect.top : rect.bottom - rect.top - BALLOON_STEMHEIGHT + 1),
```

Comment:
you say "with our GDI", but what happens if you try to test a "frankenstein ReactOS" where only the win32k/gdi32/user32/winsrv.dll (and maybe one or two other things) are changed with their Windows 2003 SP2 counterparts?

Reply:
When I tested it, it seemed that the issue was our GDI; however I may have done the Frankenstein incorrectly. When another person Frankensteined it, it seemed that the GDI did not need to be adjusted; only common controls.

Reply:
ok so here the "+ 1" in the REACTOS-specific case is still needed?

Reply:
It is still needed. I believe it's required for Windows' GDI also.